scheduler/remote_server.py

```diff
@@ -309,6 +310,9 @@ def calculate_new_last_line_params():
             duration_td = datetime.timedelta(minutes=int(scd_line['duration']))
             finish_time = scd_line['time'] + duration_td
 
+            if finish_time < time_of_interest:
+                continue
```

Comment:
so for scd_lines, they will have a duration of 0 if they are lower priority then and scheduled at the same time?